cmd/neofs-cli/modules/accounting.go

```diff
@@ -97,12 +98,8 @@ func prettyPrintDecimal(cmd *cobra.Command, decimal *accounting.Decimal) {
 		cmd.Println("value:", decimal.Value())
 		cmd.Println("precision:", decimal.Precision())
 	} else {
-		// divider = 10^{precision}; v:365, p:2 => 365 / 10^2 = 3.65
-		divider := math.Pow(10, float64(decimal.Precision()))
+		amountF8 := precision.Convert(decimal.Precision(), 8, big.NewInt(decimal.Value()))
 
-		// %0.8f\n for precision 8
-		format := fmt.Sprintf("%%0.%df\n", decimal.Precision())
-
-		cmd.Printf(format, float64(decimal.Value())/divider)
+		cmd.Println(fixedn.ToString(amountF8, 8))
```

Comment:
Why don't we just print `decimal.Value()` with original precision using `fixedn.ToString`?

Reply:
It's the point of the issue, isn't it @alexvanin?

Reply:
Sorry, I probably misunderstood the issue.